qiskit_ibm_runtime/accounts/management.py

```diff
@@ -205,7 +205,7 @@ def migrate(cls) -> None:
                     overwrite=False,
                 )
             else:
-                if hasattr(value, "auth"):
```

Comment:
Missed to find this during my review. Background: https://stackoverflow.com/a/10724829

Reply:
Without this change we would not migrate named accounts. Thus we could never remove the "auth" support from our code.